setup.py

```diff
@@ -171,7 +171,7 @@ def run(self):
 
 setuptools_args = {}
 install_requires = setuptools_args['install_requires'] = [
-    'mistune',
+    'mistune!=0.6',
```

Comment:
should not be `>0.6`? Or passing `escape=False` to previous version will work too?